packages/common-testnets/src/TestnetFactory.test.ts

```diff
@@ -122,6 +122,23 @@ describe('TestnetFactory', () => {
           const actualCode = await testnetClient.getCode({ address: randomContract })
           expect(actualCode).toEqual(newBytecode)
         })
+
+        it('sets next block base fee correctly', async () => {
+          const baseFee = 200000n
+          if (factory.constructor.name === 'TenderlyTestnetFactory') {
+            await expect(testnetClient.setNextBlockBaseFee(baseFee)).toBeRejected()
```

Comment:
Please add info why it's rejected: i think this should work:

```suggestion
            await expect(testnetClient.setNextBlockBaseFee(baseFee)). toBeRejectedWith('Method not supported')
```